src/RA_User.cpp

```diff
@@ -83,7 +83,7 @@ void LocalRAUser::RequestFriendList()
 
 RAUser* LocalRAUser::AddFriend(const std::string& sUser, unsigned int nScore)
 {
-    RAUser* pUser = RAUsers::GetUser(sUser);
+    const auto pUser = gsl::make_not_null(RAUsers::GetUser(sUser));
```

Comment:
The wrapped pointer is not a pointer to const.
Fully it would be `const gsl::not_null<RAUser*>`

Reply:
`GetUser` will allocate a user if one is not found. It's guaranteed to not return null. Would changing the signature to return "`RAUser&`" satisfy the tool?